### PR TITLE
Inconspicuous fix

### PR DESCRIPTION
**Pull request by garbear**
I mean, this bug has been around for a really long time. I blame the developers who ported XBMC away from the Xbox. Seriously, who uses Windows, Linux, OSX, Android, iOS, Raspberry Pi, FreeBSD, Fire Stick, or any Chinese box? The original linux developers should have known that the lack of games would be Kodi's longest standing bug, lasting over 9 1/2 years, and just given up. All it took to add games back was a minor fix of tens of thousands of lines of code. Some people are so lazy...

## Description
A small fix, really.

## Motivation and Context
I've been motivated by all the people who've contributed code:

* Game info tags: fetzerch for fixing a PVR-related crash
* Game add-ons: Themaister for rewind functionality, fetzerch for mouse support, file length check and cmake modifications, topfs2 for fixing a crash when loading game clients, eibma for fixing linux compilation errors, a1rwulf for catching a missing callback symbol and fixing some rebase errors, notspiff for helping with the rebrand and cmake, hudokkow for updating libretro cores, and lrusak for updating libretro cores and LibreELEC builds.
* RP: elpendor for RGB565 support, poisson for RAII improvements, ChrisMyhre for catching a compile error, notspiff for CMake fixes, acmiyaguchi for video and audio codec support, and popcornmix for Raspberry Pi support.

## How Has This Been Tested?
Lots of drunken Mario Kart.

## Types of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by fritsch** [comment]
:-)

Am 01.12.2016 12:02 vorm. schrieb "Garrett Brown" <notifications@github.com

File: system/keymaps/gamepad.xml [hunk 1]
```diff
@@ -84,6 +84,21 @@
       <dpaddown>ChapterOrBigStepBack</dpaddown>
     </gamepad>
   </FullscreenVideo>
+  <FullscreenVideo>
```
Comment (da-anda) [inline system/keymaps/gamepad.xml]: Isn't this supposed to be <FullscreenGame>?

**Comment by da-anda** [comment]
Wasn't there a plan once to call it gameplayer instead of retroplayer?

**Comment by garbear** [comment]
jenkins build this please

**Comment by Memphiz** [comment]
So funny :D

**Comment by garbear** [comment]
jenkins build this please

**Comment by garbear** [comment]
jenkins build this please

**Comment by garbear** [comment]
can someone add the "needs-backport" tag?

**Comment by ace20022** [comment]
I started to review it ;) Anyway congrats!

**Comment by FernetMenta** [comment]
@garbear you have a lot of dependency to VP in this code. As a result retroplayer is rather broken on my branch becuase VP has already progressed with lots of changes, in particular adaption to new ffmpeg api. I will make it compile on my branch. Would be nice if you could care for a fix.

**Comment by garbear** [comment]
the VP dependencies are to support game add-ons that have encoded video and audio. ATM none do, so feel free to comment out gratuitously. I could send you a PR using your new APIs, but if you're concerned about exposing VP's codecs, then we best drop codec support for now.

**Comment by garbear** [comment]
TBH I forgot we even had that feature :)

**Comment by FernetMenta** [comment]
If we don't use that know, we should drop it. I am not a friend of carrying dead code

**Comment by garbear** [comment]
well, it's used by a game addon, but the addon is still in development. what I meant is that we can break the game addon for now

**Pull request merged**